cookbook/purge/purge_version_entities.rst

```diff
@@ -0,0 +1,107 @@
+How to adapt the version purger to your needs
+=============================================
+
+After an intensive use of the PIM, it is possible that your database volume has increased in a way that the PIM holds too many unused versions.
```

Comment:
Yes but the next sentence have "it is possible" too in the other sence... It's confusing.